### PR TITLE
fix(TPS-2975, TDI-41876) Importing a job from 6.2.1 to 7.1.1 is throwing error. Clean up user data 

### PR DESCRIPTION
**Pull request by dmytroshekera**
clean up user data for cases when default values wrapped with symbols :     ' " ( )
'000'
(000)
"NULL"

### CODE REVIEW

**Comment by gonchar-ivan** [comment]
https://jira.talendforge.org/browse/TDI-41876
added link to ticket

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java [hunk 1]
```diff
@@ -238,6 +238,93 @@ else if (JavaTypesManager.STRING.getId().equals(tt) || JavaTypesManager.FILE.get
...
+            return Float.parseFloat(result);
+        }
+    }
+
+    /*
+     * don't want to corrupt user data but should check couple cases
+     * in other case return the same expression - probably it is user's expression
+     */
+    private static String cleanupUserDefaultInputs(String inStr) {
+        if (inStr.length() > 2) {
```
Comment (gonchar-ivan) [inline main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java]: Do you think this check is necessary here?
I think about following input `"" ` (2 double quotes characters). It is not empty string and not null. This input will go to Lond.parseLong method and will cause 2nd exception.

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java [hunk 2]
```diff
@@ -238,6 +238,93 @@ else if (JavaTypesManager.STRING.getId().equals(tt) || JavaTypesManager.FILE.get
...
+            }
+            return Float.parseFloat(result);
+        }
+    }
+
+    /*
+     * don't want to corrupt user data but should check couple cases
+     * in other case return the same expression - probably it is user's expression
+     */
+    private static String cleanupUserDefaultInputs(String inStr) {
```
Comment (gonchar-ivan) [inline main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java]: I would suggest following method names:
trimQuotes
cleanString
cleanInput (maybe)

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java (hunk 1 above)
Comment (dmytroshekera) [reply main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/model/metadata/MetadataToolAvroHelper.java]: empty string will cause NumberFormatException so it make no sense parse lower than 3 symbols. 
e.g. 
System.out.println(Long.parseLong(""));   // output    Exception in thread "main" java.lang.NumberFormatException: For input string: ""

**Comment by dmytroshekera** [comment]
it was quick emergency case fix. 
General fix done. So no need in this PR

**Pull request closed without merging**